eng/helix/helix.proj

```diff
@@ -23,13 +23,29 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>$(NETCoreSdkVersion)</DotNetCliVersion>    
+    <DotNetCliChannel>Current</DotNetCliChannel>
 
     <!-- Copied from Microsoft.NET.DefaultOutputPaths.targets in the .NET SDK. The Helix SDK contains nothing similar. -->
     <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">bin\</BaseOutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration)\</OutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' != 'AnyCPU'">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
   </PropertyGroup>
 
+  <!-- Specify the runtime we need which will be included as a correlation payload -->
+  <ItemGroup>
+    <!--
+      When the targeting pack builds, it has exactly the same version as the shared framework. Passing
+      SharedFxVersion because that's needed even when the targeting pack isn't building.
+      Use the BrowserDebugHost transport package as a sentinel for the non-shipping version of the NETCoreApp shared framework.
+    -->    
+    <AdditionalDotNetPackage Include="$(MicrosoftNETCoreBrowserDebugHostTransportVersion)">
```

Comment:
@dougbu doesn't look like helix.proj gets these MicrosoftNETCoreBrowserDebugHostTransportVersion (nor SharedFxVersion) which is set in the top level Directory.build.targets, is there an easy way for me to pull those into this project?  I see there's an empty Directory.build.targets in this directory is that the issue?

Reply:
Ah, that's an issue. Also a problem that this is not a .NET SDK project.

Workaround is to duplicate https://github.com/dotnet/aspnetcore/blob/main/eng/Publishing.props#L41-L56 in a target. That'll need to run prior to target you added in Helix SDK.

Reply:
Actually, just import ../Versions.props. You don't need or want `$(SharedFxVersion)` since that's the version of Microsoft.AspNetCore.App.Runtime, not Microsoft.NETCore.App.Runtime.